### PR TITLE
Reintroduce a working benchmark application

### PR DESCRIPTION
**Pull request by pricem14pc**
# What and why?

Minimal interim changes to get this working. By no means a comprehensive refactor.

### CODE REVIEW

File: _infra/helm/ras-rm-benchmark/templates/job.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ kind: Job
 metadata:
   name: {{ .Chart.Name}}
 spec:
-  ttlSecondsAfterFinished: 0
+  ttlSecondsAfterFinished: 3600
```
Comment (pricem14pc) [inline _infra/helm/ras-rm-benchmark/templates/job.yaml:6]: If the Kubernetes job is removed when it completes, the Spinnaker step will never go green (it times-out and goes red !). Leaving the completed job in the Cluster means Spinnaker can see its completed successfully and the step goes green.

File: README.md [hunk 2]
```diff
@@ -3,3 +3,33 @@ A benchmarking tool for RASRM
 
 This repository was heavily inspired by the [eq survey runner benchmark](https://github.com/ONSdigital/eq-survey-runner-benchmark).
 
+**TODO**: The following will be improved before merging and is just to capture useful commands in the interim
```
Comment (mfcrocker) [inline README.md]: Is this TODO completed?

File: _infra/helm/ras-rm-benchmark/templates/job.yaml [hunk 3]
```diff
@@ -19,8 +19,10 @@ spec:
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           env:
             - name: GOOGLE_CLOUD_PROJECT
-              value: "ras-rm-performance-tests"
+              value: "ras-rm-performance-20220908"
```
Comment (mfcrocker) [inline _infra/helm/ras-rm-benchmark/templates/job.yaml:22]: Is this correct?

File: _infra/helm/ras-rm-benchmark/values.yaml [hunk 4]
```diff
@@ -5,5 +5,9 @@ image:
   pullPolicy: Always
 
 output:
-  bucket: "ras-rm-performance-results"
+  bucket: "ras-rm-performance-20220908-locust"
```
Comment (mfcrocker) [inline _infra/helm/ras-rm-benchmark/values.yaml:8]: As above

File: _infra/helm/ras-rm-benchmark/values.yaml [hunk 5]
```diff
@@ -5,5 +5,9 @@ image:
...
   directory: "."
+
+benchmark:
+  bucket: "ras-rm-performance-20220908-benchmark"
```
Comment (mfcrocker) [inline _infra/helm/ras-rm-benchmark/values.yaml:12]: As above

**Review by mfcrocker — APPROVED** [review]
Generally looks good, couple of potential loose threads

File: _infra/helm/ras-rm-benchmark/templates/job.yaml (hunk 3 above)
Comment (JacobHarrisonONS) [reply _infra/helm/ras-rm-benchmark/templates/job.yaml:22]: yes 
https://github.com/ONSdigital/ras-rm-performance-tests/pull/22/files#:~:text=GOOGLE_CLOUD_PROJECT%3A%20%22ras%2Drm%2Dperformance%2D20220908,%22ras%2Drm%2Dperformance%2D20220908%2Dlocust%22

File: README.md (hunk 2 above)
Comment (pricem14pc) [reply README.md]: Updated

File: _infra/helm/ras-rm-benchmark/values.yaml (hunk 5 above)
Comment (pricem14pc) [reply _infra/helm/ras-rm-benchmark/values.yaml:12]: As above, this is the correct project

File: _infra/helm/ras-rm-benchmark/values.yaml (hunk 4 above)
Comment (pricem14pc) [reply _infra/helm/ras-rm-benchmark/values.yaml:8]: This is the correct project

**Pull request merged**